Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -335,8 +335,9 @@ extension BrowserViewController: WKUIDelegate {
                  initiatedByFrame frame: WKFrameInfo,
                  type: WKMediaCaptureType,
                  decisionHandler: @escaping (WKPermissionDecision) -> Void) {
-        // If the tab isn't the selected one, do not show the media capture prompt
-        guard tabManager.selectedTab?.webView == webView else {
+        // If the tab isn't the selected one or we're on the homepage, do not show the media capture prompt
+        let hasNoHomepage = CoordinatorFlagManager.isCoordinatorEnabled ? !contentContainer.hasHomepage: homepageViewController?.view.alpha == 0
```

Comment:
Seems like this line is too long. Wondering if we can clean this up a bit to be more readable. Adding a suggestion for your reference

```suggestion
        let isCordinatorEnabled = CoordinatorFlagManager.isCoordinatorEnabled
        let noContainerHomepage = !contentContainer.hasHomepage
        let homepageControllerHidden = homepageViewController?.view.alpha == 0
        let hasNoHomepage = isCordinatorEnabled ? noContainerHomepage : homepageControllerHidden
```

Reply:
We cannot have the code run if the feature flag isn't enabled for it, I can use a `if` `else` if you want, but cannot use this change here.

Reply:
Since you will be removing this anyways, you can merge the PR. I will follow-up in a dm for a separate explanation regarding `We cannot have the code run if the feature flag isn't enabled for it`